fail fast if user input parameters invalid/not exist

today I see `access-denied` error instead of input validation:
```
docker run --rm -it --network=host   --mount type=bind,source=/mnt,target=/mnt   --mount type=bind,source=/etc/os-release,target=/etc/os-release   --mount type=bind,source=${DHCLIENT_LEASE_FILE},target=/var/lib/dhclient/dhclient.leases   ghcr.io/opiproject/opi-sztp-client:v0.2.0   /opi-sztp-agent daemon --bootstrap-trust-anchor-cert /mnt/nonexist.pem --device-end-entity-cert /mnt/nonexist.pem --device-private-key /mnt/nonexist.pem --serial-number opi-serial-number
Run the daemon command

Usage:
  opi-sztp-agent daemon [flags]

Flags:
      --bootstrap-trust-anchor-cert string   Bootstrap server trust anchor Cert (default "/certs/opi.pem")
      --device-end-entity-cert string        Device's End Entity cert (default "/certs/my_cert.pem")
      --device-password string               Device's password (default "my-secret")
      --device-private-key string            Device's private key (default "/certs/private_key.pem")
      --dhcp-lease-file string               Device's dhclient leases file (default "/var/lib/dhclient/dhclient.leases")
  -h, --help                                 help for daemon
      --serial-number string                 Device's serial number (default "my-serial-number")
2024/06/17 13:04:09 [INFO] Get the Bootstrap URL from DHCP client
2024/06/17 13:04:09 [INFO] Bootstrap URL retrieved successfully: https://bootstrap:8080/restconf/operations/ietf-sztp-bootstrap-server:get-bootstrapping-data
2024/06/17 13:04:09 [INFO] Starting the Request to get On-boarding Information.
2024/06/17 13:04:09 [DEBUG] Sending to: https://bootstrap:8080/restconf/operations/ietf-sztp-bootstrap-server:get-bootstrapping-data
2024/06/17 13:04:09 [DEBUG] Sending input: {"ietf-sztp-bootstrap-server:input":{"hw-model":"BlueField SoC","os-name":"Ubuntu 22.04.4 LTS","os-version":"22.04","nonce":""}}
2024/06/17 13:04:09 [ERROR]  [ERROR] Expected conveyed-information, received error type=protocol, tag=access-denied, message=
Error: [ERROR] Expected conveyed-information, received error type=protocol, tag=access-denied, message=

```

--- a/sztp-agent/cmd/daemon.go
+++ b/sztp-agent/cmd/daemon.go
@@ -9,6 +9,9 @@ Copyright (C) 2022 Red Hat.
 package cmd
 
 import (
+	"fmt"
+	"os"
+
 	"github.com/opiproject/sztp/sztp-agent/pkg/secureagent"
 	"github.com/spf13/cobra"
 )
@@ -29,6 +32,14 @@ func NewDaemonCommand() *cobra.Command {
 		Use:   "daemon",
 		Short: "Run the daemon command",
 		RunE: func(c *cobra.Command, args []string) error {
+			arrayChecker := [4]string{dhcpLeaseFile, devicePrivateKey, deviceEndEntityCert, bootstrapTrustAnchorCert}
+			for _, filePath := range arrayChecker {
+				info, err := os.Stat(filePath)
+				cobra.CheckErr(err)
+				if info.IsDir() {
+					return fmt.Errorf("must not be folder: %q", filePath)
+				}
+			}
 			err := c.Help()
 			cobra.CheckErr(err)
 			a := secureagent.NewAgent(bootstrapURL, serialNumber, dhcpLeaseFile, devicePassword, devicePrivateKey, deviceEndEntityCert, bootstrapTrustAnchorCert)

